lib/v8debugapi.js

```diff
@@ -149,7 +150,8 @@ module.exports.create = function(logger_, config_, fileStats_) {
                 breakpoint.location.line = mappedPos.line;
               } catch (err) {
                 return setErrorStatusAndCallback(cb, breakpoint,
-                  StatusMessage.CONDITION, messages.SOURCE_MAP_READ_ERROR + err);
+                  StatusMessage.BREAKPOINT_CONDITION,
```

Comment:
In that case, it sounds like we are missing a test-case. Can you add a test-case for this and other error returns from this function?  separate PR is fine.